flatdict.py

```diff
@@ -2,15 +2,18 @@
 key/value pair mapping of nested dictionaries.
 
 """
-import collections
+try:
+  from collections.abc import MutableMapping
```

Comment:
Is not PEP8 compliant - and flake8 tests failed. Please correct so it can be merged.